servicemanager.go

```diff
@@ -101,26 +108,19 @@ func init() {
 func cleanup(cfg *config.Config, dbFile string) {
 	log.Info("System cleanup")
 
-	if err := launcher.Cleanup(cfg); err != nil {
-		log.Errorf("Can't cleanup launcher: %s", err)
+	if err := os.RemoveAll(cfg.ServicesDir); err != nil {
```

Comment:
In this commit PR https://github.com/aoscloud/aos_servicemanager/pull/30, the following code is added:
```go
	if layermanager.layersDir == "" {
		layermanager.layersDir = path.Join(config.WorkingDir, LayerDirName)
	}

```
I've asked @MykolaSolyanko to do the same for servicemanager. So, it will not work in case services and layers dirs are not defined in config.